README.md

```diff
@@ -30,7 +30,7 @@ specific Jakarta version of ESAPI, in Maven, you would specify your ESAPI depend
 <dependency>
     <groupId>org.owasp.esapi</groupId>
     <artifactId>esapi</artifactId>
-    <version>2.5.3.0-SNAPSHOT</version>
```

Comment:
I don't want to have yet another place where we need to keep the ESAPI version current in the README.md file, so instead of 2.5.4.0, let's just drop the "-SNAPSHOT" and then somewhere add a note that says that the version has to be 2.5.3.0 _or later_. That way, we won't have to bother keeping that version up-to-date.